README.md

```diff
@@ -1 +1,180 @@
-# styra-ansible-collection
\ No newline at end of file
+# Ansible Collection - styra.opa
+
+**TODO: update these instructions so that the README applies when someone installs the collection from galaxy, as opposed to writing a playbook in the root directory.**
+
+
+## Overview
+A **Policy Decision Point (PDP)** is a piece of software that when asked for a policy decision will evaluate a policy against the query and return a response.  It is designed to **decouple policy** from an underlying piece of software so that policies (which are so crucial to security, compliance, and operations) **can be managed separately from the software that enforce those policies**.
+
+[Open Policy Agent (OPA)](https://www.openpolicyagent.org/) is a **policy decision point created by Styra** and owned by the Cloud Native Computing Foundation.  It has attained the highest maturity level of CNCF projects ("graduated"), which is the same as maturity level as Kubernetes, Prometheus, Envoy, etc.
+
+
+OPA is a lightweight PDP and is designed to distribute the load of policy decision making as close to the edge of a software application or infrastructure as possible.  By running OPA near the edge, policy decisions achieve high performance and high availability by a shared-fate architecture.
+
+Styra also provides a **commercial-grade management plane for OPA** that implements a **single pane of glass** for controlling the many instances of OPA that are often required for a full policy solution.  This product called [Styra DAS](https://www.styra.com/styra-das/) provides
+* full **policy lifecycle management** (authoring, testing, deploying, monitoring, and logging)
+* **enterprise governance** (global policies that span many teams)
+* **specialized support for common OPA use cases** (kubernetes, terraform IaC, envoy-based service meshes, kong gateway)
+
+This ansible collection provides functionality to install OPA as a standlone PDP.  (Often OPA is run as a sidecar as part of a Pod instead of as a standalone service.  Such an installation is out of scope as it requires modifying for example k8s Deployments that make up the application.) There are 3 possible configuration options supported by this ansible collection:
+
+* **barebones**:  No usage of any of the [OPA management configuration](https://www.openpolicyagent.org/docs/latest/management-introduction/) settings.  You are responsible for pushing policies and data into OPA so that it can make decisions.  Not recommended for production use.
+* **bundle**: Includes [bundle configuration](https://www.openpolicyagent.org/docs/latest/management-bundles/) so that policy/data bundles are served from an HTTP server.
+* **das_discovery**: Includes [discovery configuration](https://www.openpolicyagent.org/docs/latest/management-discovery/), which downloads bundle/status/decision-log configuration and then applies it.  Specifically tailored to [Styra DAS](https://www.styra.com/styra-das/).
+
+
```

Comment:
Add the a section on requirements:
- Ansible versions
- Dependencies (if relevant)
- Python versions

Reply:
Done